src/IB/IBFESurfaceMethod.cpp

```diff
@@ -327,9 +341,33 @@ IBFESurfaceMethod::preprocessIntegrateData(double current_time, double new_time,
     d_F_half_vecs.resize(d_num_parts);
     d_F_IB_ghost_vecs.resize(d_num_parts);
 
-    d_DP_systems.resize(d_num_parts);
-    d_DP_half_vecs.resize(d_num_parts);
-    d_DP_IB_ghost_vecs.resize(d_num_parts);
+    d_P_j_systems.resize(d_num_parts);
```

Comment:
What does `j` stand for? (I understand `i` and `o`.)

Reply:
so j is the jump. i and o are interior and exterior? If it's not intuitive enough we can change it to something else?

Reply:
Let me think. `j` makes me think of an index.